tools/wpt/testfiles.py

```diff
@@ -341,7 +341,8 @@ def affected_by_interfaces(file_contents):
 
 def get_parser():
     # type: () -> argparse.ArgumentParser
-    parser = argparse.ArgumentParser()
+    parser = argparse.ArgumentParser(epilog="This command supports python3 and thus could be "
```

Comment:
I would really like us to develop a way of annotating this in the command.json files rather than hardcoding it in the description for specifc commands. Can we revert this part until that's done?

Reply:
OK, I had a similar idea. Perhaps we should develop it before going to far in the migration. I'll revert.